packages/social-previews/README.md

```diff
@@ -87,6 +89,36 @@ const tweetTemplate = {
 />;
 ```
 
+An example of LinkedIn preview
+
+```jsx
+import { LinkedInPreview } from '@automattic/social-previews';
+
+<LinkedInPreview
+	jobTitle="Job Title (Company Name)"
+	image="https://url.for.the/image.png"
+	name="Linked Account Name"
```

Comment:
I don't think it really matters, but was this supposed to be `LinkedIn Account Name`?

Reply:
Fixed. Thanks